README.md

```diff
@@ -1,22 +1,7 @@
-# BetterPenetration
-This plugin seeks to solve to "telescoping" issue that occurs in HScenes in AI Shoujo and Honey Select 2.  It also adds some additional features to overall improve the HScene experience.  This plugin does not work for studio.
+Fork of https://github.com/Animal42069/BetterPenetration
 
-# Features
```

Comment:
I don't think the pull request should wipe out the entire readme... sure some of it is outdated but not all of it

Reply:
Lol ! Sorry, I updated the Readme on my repository and I didn't think about the fact it would be added to the pull request.
Can you exclude the Readme file from the pull request on your end, or do you want me to undo the commit on my branch?

Reply:
I've reverted the changes made to the Readme file, you should be able to merge now. Thanks